src/stories/PayPalButtons.stories.js

```diff
@@ -93,17 +93,23 @@ StandAlone.parameters = {
 
 function DynamicAmountTemplate(args) {
     const [amount, setAmount] = useState(2);
+    const [orderID, setOrderID] = useState();
```

Comment:
I think it would be nice to have a default value for Order ID and reset it when the amount changes.
```suggestion
    const [orderID, setOrderID] = useState('unknown');
```

Reply:
@gregjopa Why not set the default value to something falsy rather than `unknown`?

Reply:
@mnicpt good call. Setting it to false seems like a better design. Then we can conditionally show the "unknown" label in jsx like so:
```jsx
<p>Order ID: {orderID ? orderID : 'unknown' }</p>
```